mozci/task.py

```diff
@@ -273,6 +276,22 @@ def configuration(self):
             get_test_variant(self.variant, self.label),
         )
 
+    @property
+    def is_potential_regression(self):
+        # Check if the job is either tier 1 or 2
+        if self.tier not in (1, 2):
+            return False
+
+        # Check if job result is busted or exception
+        if self.result not in ("busted", "exception"):
+            return False
+
+        # Check if extra.treeherder.jobKind is build
+        if self.job_kind != "build":
+            return False
+
+        return True
```

Comment:
Same as https://github.com/mozilla/mozci/pull/1148#discussion_r2025690780, we should move this in mozci/push.py